gadgets/moco/config/cpureg_cartesian_averaging.xml

```diff
@@ -2,67 +2,49 @@
 <gadgetronStreamConfiguration xsi:schemaLocation="http://gadgetron.sf.net/gadgetron gadgetron.xsd"
```

Comment:
We can remove all the Cartesian to generic stuff. The Gadget, the XMLs, any tests that use it.